addons/mail/static/src/discuss/core/public_web/discuss_category_model.js

```diff
@@ -0,0 +1,33 @@
+import { fields, Record } from "@mail/core/common/record";
+
+export class DiscussCategory extends Record {
+    static _name = "discuss.category";
+    static id = "id";
+    static _inherits = { DiscussAppCategory: "appCategory" };
```

Comment:
I don't think we should use that, it was mainly introduced to mimic python models.

I'd rather we explicitly follow the relation and write the the correct fields whenever necessary.

What is it used for?